src/Features/Core/Portable/Completion/Providers/SymbolCompletionItem.cs

```diff
@@ -208,10 +206,10 @@ private static Document FindAppropriateDocumentForDescriptionContext(Document do
                 }
             }
 
-            return contextDocument;
+            return contextDocument ?? document;
```

Comment:
I wonder if this would cover up some bug in the future, since we do expect `GetDocument(contextId)` above to return non-null. Maybe immediately return the value from a call to `GetRequiredDocument(contextId)` in the if-statement instead?